ui/packages/ace/.nsprc

```diff
@@ -0,0 +1,5 @@
+{
+  "exceptions": [
+    "https://nodesecurity.io/advisories/566"
```

Comment:
❓ Can we leave a comment to why we added exceptions?

Reply:
@pklinef  Good question. I asked the same thing. I'm not aware of way to add comments in JSON particularly for this format. I've only seen hacky ways of making them a field/value. But I would like to add it if we are aware of a way to do it. I wonder if we can track of it elsewhere if not directly in this file.

Reply:
@pklinef I tried it out again, but remember why now we couldn't. When I add a comment it is like the file fails to get parsed and the exceptions are no longer read. Looks like comments worked at one point and doesn't anymore: https://github.com/nodesecurity/nsp/issues/207#issuecomment-378594564